### ISSUE TITLE
can't evaluate field HasTokens in type *config.CompactAuthTokens

### ISSUE THREAD

**Issue by jollinshead** [kubernetes-incubator/kube-aws#469]
If we set 
```
ManageCertificates: false
``` 
the method ```HasTokens``` is not resolvable to evaluate the expression 
```
{{ if .AuthTokensConfig.HasTokens }}
```
since the ```AuthTokensConfig``` type is not initialised.

_Error message:_
```
executing "userdata/cloud-config-controller" at <.AuthTokensConfig.Ha...>: can't evaluate field HasTokens in type *config.CompactAuthTokens
```

**Suggested fix**: Initialise ```AuthTokensConfig``` beforehand. https://github.com/kubernetes-incubator/kube-aws/pull/470

**Comment by jollinshead** [comment]
https://github.com/kubernetes-incubator/kube-aws/pull/470

**Comment by mumoshu** [comment]
I"m closing this as #470 is already merged and included in v0.9.5.
Please feel free to reopen if the problem still persists.
Thanks for your support @jollinshead :bow:
